localnet/poktrolld/config/relayminer_config.yaml

```diff
@@ -1,19 +1,19 @@
-signing_key_name: supplier1
+signing_key_name: shannon_supplier
```

Comment:
Make sure `shannon_supplier` signing key name exists in `config.yml` and that it has a stake target in `Makefile`